src/material/table/table.scss

```diff
@@ -25,6 +43,57 @@
   // The MDC table does not allow text to wrap within the cell. This allows for text to
   // wrap when the cell reaches its maximum width.
   white-space: normal;
+
+  @include token-utils.use-tokens(tokens-mat-table.$prefix, tokens-mat-table.get-token-slots()) {
+    @include token-utils.create-token-slot(background-color, background-color);
+  }
+}
+
+@include mdc-helpers.disable-mdc-fallback-declarations {
+  @include token-utils.use-tokens(tokens-mat-table.$prefix, tokens-mat-table.get-token-slots()) {
+    .mat-mdc-header-row {
```

Comment:
I had to leave the fallback values for some tokens to reduce the amount of screenshot failures. Will try to clean them up in a follow up.